src/Blocks/Block.php

```diff
@@ -148,7 +148,13 @@ protected static function parse_attributes($data, $block_type) {
 
 			$validator = new Validator();
 
-			$result = $validator->schemaValidation((object) $attributes, $schema);
+			if (empty($data['innerHTML'])) {
+				$data['innerHTML'] = '<span data-warning="This block does not contain a render template"></span>';
```

Comment:
Could we leave there just an empty span and log this message with `error_log` and `E_WARNING` ?

Reply:
Could do, but would need to conditionally call `source_attributes` with `HtmlDomParser` on L16, as that's what's causing the error. The DOM parser function error'so out when an empty string is passed in.

Reply:
I'd also say, having played around, I'm not entirely sure about what `innerHTML` is, in an ACF defined Gutenberg block, versus a WP core block. And whether it's supposed to contain the HTML from render Template, do you know?